site-container.yml.sample

```
@@ -517,7 +517,7 @@
       run_once: true
       when: not ceph_status.failed
 
-- hosts: '{{ groups["grafana-server"][0] | default(groups["mgrs"][0]) | default(groups["mons"][0]) | default(omit) }}'
+- hosts: '{{ groups["grafana-server"] | default(groups["mgrs"]) | default(groups["mons"]) | default(omit) }}'
```

according to last discussion, we should have only grafana-server
```suggestion
- hosts: '{{ groups["grafana-server"] | default(omit) }}'
```

---

done